src/test/java/uk/gov/companieshouse/mapper/OtherReasonMapperTest.java

```diff
@@ -17,7 +19,12 @@
 
 @ExtendWith(SpringExtension.class)
 public class OtherReasonMapperTest {
-    private final OtherReasonMapper mapper = new OtherReasonMapper(new AttachmentMapper());
+
+    @Spy
```

Comment:
We could mock and use verifies here to make this class more consistent with the others.

Reply:
Thanks Jonathan! Improvement done